### PR TITLE
Reenable BF16 tests on AX2 machines

### PR DESCRIPTION
**Pull request by rengolin**

### CODE REVIEW

File: test/BF16/Integration/lit.local.cfg [hunk 1]
```diff
@@ -20,5 +20,5 @@ def has_support(feature):
     return True
 
 # AVX512 and SVE should have BF16 support
-if not has_support('avx512') and not has_support('sve'):
+if not has_support('avx512') and not has_support('avx2') and not has_support('sve'):
```
Comment (alheinecke) [inline test/BF16/Integration/lit.local.cfg:23]: SVE has the same support as AVX512. in libxsmm.. not sure why it's excluded

**Review by alheinecke — CHANGES_REQUESTED** [review]

File: test/BF16/Integration/lit.local.cfg (hunk 1 above)
Comment (alheinecke) [reply test/BF16/Integration/lit.local.cfg:23]: sse and avx should be excluded

Comment (rengolin) [reply test/BF16/Integration/lit.local.cfg:23]: It's not excluded. This is a double negative. It is "supported" is there's AVX2, AVX512 or SVE.

Comment (rengolin) [reply test/BF16/Integration/lit.local.cfg:23]: > sse and avx should be excluded

It is. These tests are **only** run on AVX2, AVX512 and SVE.

**Comment by rengolin** [comment]
For the record, the CI machine was already running the BF16 tests, so no changes were visible on the number of "Unsupported" tests. However, on my ageing desktop I used to get 35 unsupported and now I get 21 (like the CI machine) and all tests pass.

**Review by alheinecke — APPROVED** [review]

**Comment by alheinecke** [comment]
yes it should be fine since April, when the compiler bug was fixed

**Comment by hfp** [comment]
> For the record, the CI machine was already running the BF16 tests, so no changes were visible on the number of "Unsupported" tests. However, on my ageing desktop I used to get 35 unsupported and now I get 21 (like the CI machine) and all tests pass.

( The `tpp-gpu` pipeline or the Intel based tests are running on a RaptorLake system and we should see the change there as well. )

**Pull request merged**